### ISSUE TITLE
regression seneca >= 3.20.4 broke plugins

### ISSUE THREAD

**Issue by wzrdtales** [senecajs/seneca#792]
When upgrading to 3.20.4 plugins are broken from this point on:

```
he definition action for the plugin mesh has failed: Cannot read property 'length' of undefined (api.js:300:28). This error is considered fatal as all plugins have to initialize correctly. You should test the plugin by itself to verify that it is working correctly. Also ensure that the configuration options passed to the plugin are correct. These are shown below under in the DETAILS section. There could also be a bug in the plugin. If you think that is the case, please create a github issue on the plugin's repository, and include this error report.","code":"plugin_define_failed","err":{"eraro":true,"orig":{},"code":"plugin_define_failed","seneca":true,"package":"seneca","msg":"seneca: The definition action for the plugin mesh has failed: Cannot read property 'length' of undefined (api.js:300:28). This error is considered fatal as all plugins have to initialize correctly. You should test the plugin by itself to verify that it is working correctly. Also ensure that the configuration options passed to the plugin are correct. These are shown below under in the DETAILS section. There could also be a bug in the plugin. If you think that is the case, please create a github issue on the plugin's repository, and include this error report.","details":{"fullname":"mesh","message":"Cannot read property 'length' of undefined (api.js:300:28)","options":{"bases":["127.0.0.1:39000","127.0.0.1:39001"],"host":"127.0.0.1","sneeze":{"silent":"true"},"monitor":"false","auto":true,"listen":[{"pin":"service:test,command:*","model":"consume"}],"discover":{"rediscover":true,"custom":{"active":true}}},"repo":"","errmsg":"Cannot read property 'length' of undefined","errline":"at Seneca.exports.depends
```

**Comment by rjrodger** [comment]
Thank You!

**Comment by rjrodger** [comment]
fixed in 3.20.6
